src/public/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" href="style.css">
+    <h1> Yoooooo </h1>
+  </head>
+    <body>
+      <h2> Y'all ready for this?</h2>
```

Comment:
Minor, but you should always begin with `<h1>` in an HTML section. If the text requires multiple heading levels, work your way up. You can target and style specific heading levels inside specific sections using CSS. Having good semantic markup can help search engines correctly crawl the site and generate previews.